### DISCUSSION TITLE
Mapserver - Validate?

### DISCUSSION THREAD

**Discussion by amigators** [Homebrew/discussions#2483] [Getting started]
Mac OS 11.6.1
Brew install of Mapserver latest version

Trying to access map file get the following message:
msCGILoadMap(): Web application error. CGI variable "map" fails to validate.

Read up a bit might need to use SetEnv. in the http.conf file?

Documentation points to 
SetEnv MS_MAP_PATTERN "^\/opt\/mapserver\/([^\.][_A-Za-z0-9\-\.]+\/{1})*([_A-Za-z0-9\-\.]+\.(map))$"

in my brew install I can not find a /opt/mapserver/. directory... 
There is a lot I do not know.

Thanks.

**Comment by Losrity** [comment]
i have same question And little help for you:https://www.osgeo.cn/post/1fb97
but i can't understand this
